### DISCUSSION TITLE
Need help with some config

### DISCUSSION THREAD

**Discussion by caprinux** [kovidgoyal/kitty#4045] [Q&A]
![image](https://user-images.githubusercontent.com/76640319/134117578-98684ed2-7a6a-42e1-ae90-d3f1dd291fb1.png)

How does anyone have any idea on how I can scale the tint of my terminal, such that there aren't gaps in between each split where I can see 100% of my background image? Thanks

**Comment by kovidgoyal** [comment]
Use 

background_opacity 0 
background_tint 0.7

**Reply by kovidgoyal** [reply]
Or actually it possibly wont do what you need since tinting is applied only uder the text area. Instead just darken the background image itself.

**Reply by rag-hav** [reply]
> Or actually it possibly wont do what you need since tinting is applied only uder the text area. Instead just darken the background image itself.

would love if this could be configured
